external_contact.go

```diff
@@ -81,3 +85,72 @@ func (c *WorkwxApp) MarkExternalContactTag(userID, externalUserID string, addTag
 	})
 	return err
 }
+
+// ExternalContactUnassigned 离职成员的客户
+type ExternalContactUnassigned struct {
+	// HandoverUserID 离职成员的userid
+	HandoverUserID string `json:"handover_userid"`
+	// ExternalUserID 外部联系人userid
+	ExternalUserID string `json:"external_userid"`
+	// DimissionTime 成员离职时间
+	DimissionTime time.Time `json:"dimission_time"`
```

Comment:
Apparently Tencent used archaic English here: [dimission](https://www.merriam-webster.com/dictionary/dimission) vs [demission](https://www.merriam-webster.com/dictionary/demission).

From quick Google searches "demission" has far greater number of hits (23300000 vs 384000), so what about replacing this particular word?

NOTE: Not doing so is also absolutely OK for me. Users of this library should be overwhelmingly Chinese and they really want to just search the Tencent name and expect to see results.

Reply:
i think so too, and if Tencent spell wrong word, i will fix it as possible as i can like `StatusChangeEvent string xml:"StatuChangeEvent"`  `Notifier string xml:"Notifyer"`  and so on